<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Experiment settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="_imageHolder">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Experient settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1" colspan="2">
          <widget class="QuantityEdit" name="_energy">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Energy:</string>
           </property>
           <property name="buddy">
            <cstring>_energy</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Calibrant:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="_wavelengthUnit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Ã…</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Wavelength:</string>
           </property>
           <property name="buddy">
            <cstring>_wavelength</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QuantityEdit" name="_wavelength">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="_energyUnit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>keV</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Detector:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="DetectorSelector" name="_detector"/>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="CalibrantSelector" name="_calibrant"/>
         </item>
        </layout>
        <zorder>_energy</zorder>
        <zorder>label_3</zorder>
        <zorder>_wavelengthUnit</zorder>
        <zorder>_energyUnit</zorder>
        <zorder>_wavelength</zorder>
        <zorder>label_7</zorder>
        <zorder>label_6</zorder>
        <zorder>label_8</zorder>
        <zorder>_detector</zorder>
        <zorder>_calibrant</zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Acquisition</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="2" column="4">
          <widget class="QLabel" name="_imageSizeUnit">
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Image:</string>
           </property>
           <property name="buddy">
            <cstring>_image</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="_image">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QToolButton" name="_imageLoader">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="_imageSizeLabel">
           <property name="text">
            <string>Image size:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="_imageSize">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_76">
           <property name="text">
            <string>Dark file:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="_dark">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QToolButton" name="_darkLoader">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="_nextStep">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Next &gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QuantityEdit</class>
   <extends>QLineEdit</extends>
   <header>pyFAI.gui.calibration.QuantityEdit</header>
  </customwidget>
  <customwidget>
   <class>CalibrantSelector</class>
   <extends>QComboBox</extends>
   <header>pyFAI.gui.calibration.CalibrantSelector</header>
  </customwidget>
  <customwidget>
   <class>DetectorSelector</class>
   <extends>QComboBox</extends>
   <header>pyFAI.gui.calibration.DetectorSelector</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
